response_operations_ui/templates/collection_exercise/ce-eq-instrument-section.html

```diff
@@ -8,6 +8,11 @@
 {% from "components/table/_macro.njk" import onsTable %}
 {% from "components/upload/_macro.njk" import onsUpload %}
 
+{% block head %}
```

Comment:
This has to go in the extending template, because of that it can't go a level down into the import. Because of this I have had to add an If to make sure it's not the view page. Not ideal, but I can't realistically restructure in this story